dashboard/src/components/Config/AppRepoList/AppRepoRefreshAllButton.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+
+import "./AppRepo.css";
+
+interface IAppRepoRefreshAllButtonProps {
+  error?: Error;
+  resyncAllRepos: () => void;
+  kubeappsNamespace: string;
+}
+
+export class AppRepoRefreshAllButton extends React.Component<IAppRepoRefreshAllButtonProps> {
+  public render() {
+    return (
+      <span>
+        <button
+          className="button button-primary margin-l-big"
+          onClick={this.handleResyncAllClick()}
+          title="Refresh All App Repositories"
+        >
+          Refresh All
+        </button>
+      </span>
+    );
+  }
+
+  private handleResyncAllClick() {
+    return () => {
```

Comment:
I tried to use 'disabled' property of the button to avoid multiple clicks, but got problem to set/remove 'disabled' attribute.

Reply:
let's leave it for now without showing progress